nevergrad/functions/utils.py

```diff
@@ -27,30 +26,3 @@ def __call__(self, x: np.ndarray) -> np.ndarray:
         if self.rotation_matrix is not None:
             y = self.rotation_matrix.dot(y)
         return y
-
-
-class PostponedObject(abc.ABC):
```

Comment:
Both those classes are not necessary anymore since the methods are directly included in `ExperimentFunction`